components/persistor/HISTORY.md

```diff
@@ -1,4 +1,6 @@
 ## 9.0.2
```

Comment:
I believe you meant for this to be 9.1.0

Reply:
Good catch - fixed it.